src/App/Pages/Settings/VaultSettingsPageViewModel.cs

```diff
@@ -40,7 +40,7 @@ public VaultSettingsPageViewModel()
 
         private async Task GoToImportItemsAsync()
         {
-            var webVaultUrl = _environmentService.GetWebVaultUrl();
+            var webVaultUrl = _environmentService.GetWebVaultUrl() + "/#/tools/import";
```

Comment:
@djsmith85 @aj-rosado  we have a file for [External link constants](https://github.com/bitwarden/mobile/blob/main/src/Core/ExternalLinksConstants.cs#L3) so we have them in one place. I think we should change this to follow the same pattern as in other places like adding the formatted string in the external links constants file:
```c#
public const string WEB_VAULT_TOOLS_IMPORT_FORMAT = "{0}/#/tools/import";
```
and change it here as well:
```c#
var webVaultUrl = string.Format(ExternalLinksConstants.WEB_VAULT_TOOLS_IMPORT_FORMAT, _environmentService.GetWebVaultUrl());
```

Reply:
Aww cheers @fedemkr. Thanks for the info and I've created https://github.com/bitwarden/mobile/pull/2939 as a followup to address that.

Reply:
Awesome thank you! 👍